XMonad/Actions/CopyWindow.hs

```diff
@@ -76,18 +76,23 @@ import qualified XMonad.StackSet as W
 -- "XMonad.Doc.Extending#Editing_key_bindings".
 
 -- $logHook
--- To distinguish workspaces containing copies of the focused window use
--- something like:
---
--- > sampleLogHook h = do
--- >    copies <- wsContainingCopies
--- >    let check ws | ws `elem` copies = pad . xmobarColor "red" "black" $ ws
--- >                 | otherwise = pad ws
--- >    dynamicLogWithPP myPP {ppHidden = check, ppOutput = hPutStrLn h}
--- >
--- > main = do
--- >    h <- spawnPipe "xmobar"
--- >    xmonad def { logHook = sampleLogHook h }
+--
+-- To distinguish workspaces containing copies of the focused window, use 'copiesPP'.
+-- 'copiesPP' takes a pretty printer and makes it aware of copies of the focused window.
+-- It can be applied when creating a @StatusBarConfig@ from "XMonad.Hooks.StatusBar".
+--
+-- A sample config looks like this:
+--
+-- > mySB = statusBarProp "xmobar" (copiesPP (pad . xmobarColor "red" "black") xmobarPP)
+-- > main = xmonad $ withEasySB mySB defToggleStrutsKey def
+
+-- | Takes a pretty printer and makes it aware of copies.
```

Comment:
"Takes a pretty printer and makes it aware of copies by using the provided function to show hidden workspaces that contain copies of the focused window." might be a better doc here.